app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,10 @@
+class MoviesController < ApplicationController
+
+  def show
```

Comment:
I wonder if some of these could be done in the view to lighten up the controller a bit